core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -394,7 +394,8 @@ class BrokerServer(
 
       // TODO : add the config max.incremental.share.fetch.session.cache.slots and the variable MIN_INCREMENTAL_SHARE_FETCH_SESSION_EVICTION_MS
       //  which should be used to initialize shareFetchSessionCache
-      val shareFetchSessionCache : ShareSessionCache = new ShareSessionCache(0, 0)
+      val shareFetchSessionCache : ShareSessionCache = new ShareSessionCache(config.maxIncrementalFetchSessionCacheSlots,
```

Comment:
The comment above should be removed. The KIP says the maximum size is `group.share.max.groups * group.share.max.size`. The eviction timeout has no specific configuration for now. The KIP says 120,000 milliseconds (2 minutes).

Reply:
Thanks for pointing it our, made this change in the latest commit